wagtail_localize_rws_languagecloud/templates/wagtail_localize/admin/edit_translation.html

```diff
@@ -0,0 +1,29 @@
+{% extends "wagtailadmin/generic/edit.html" %}
```

Comment:
A couple of comments on this template.

1. Overriding the template from `wagtail_localize` like this makes the order of `INSTALLED_APPS` important. You have to specify

    ```py
    INSTALLED_APPS = [
        # ...
        'wagtail_localize_rws_languagecloud',
        'wagtail_localize',
        # ...
    ]
    ```
    
    If they're the other way round, it will use the default one from `wagtail_localize`. We should
    
    - update the README to show this in the setup
    - call this out as a possible breaking change in the changelog
    
    similar to what you've already done with https://github.com/zerolab/wagtail-localize-rws-languagecloud/pull/35#discussion_r830979939

2. Django now supports recursive template overrides so we don't need to override the entire template. We can just write 

    ```django
    {% extends "wagtail_localize/admin/edit_translation.html" %}
    
    {% block extra_css %}
       <!-- extra CSS in here -->
    {% endblock %}
    ```
    
    and that will work. The less we can override the better as if we override blocks we don't need to, any changes to the template in `wagtail_localize` will need to be mirrored here each time the upstream package changes.

Reply:
I've fixed both of these issues. Thanks for pointing them out!

I updated the README install instructions and also the CHANGELOG. I didn't think it made sense to put breaking change alerts in the CHANGELOG.md file though, so I'm hoping we can just put that in the release notes.

I updated the template as you mentioned. I wasn't aware of recursive template overrides. TIL!